sumOfOther.js

```diff
@@ -0,0 +1,10 @@
+function sumOfOther(arr) {
+  let newArr = [];
+  let buf;
+  let red = arr.reduce(function(x,y) { return x + y; });
```

Comment:
@GoldenKit 'red' is never reassigned. Use 'const' instead. Writing code on one line is a bad practice.